plugins/ratio/ratio.php

```diff
@@ -159,7 +159,7 @@ public function flush()
 					$req->addCommand( rTorrentSettings::get()->getRatioGroupCommand("rat_".$i,'ratio.enable',array("")) );
 					$req->addCommand( rTorrentSettings::get()->getRatioGroupCommand("rat_".$i,'ratio.min.set',$rat["min"]) );
 					$req->addCommand( rTorrentSettings::get()->getRatioGroupCommand("rat_".$i,'ratio.max.set',$rat["max"]) );
-					$req->addCommand( rTorrentSettings::get()->getRatioGroupCommand("rat_".$i,'ratio.upload.set',floatval($rat["upload"]*1024*1024)) );
+					$req->addCommand( rTorrentSettings::get()->getRatioGroupCommand("rat_".$i,'ratio.upload.set',floatval($rat["upload"]*1024*1024*1024)) );
```

Comment:
What about compatibility with an old data?
For example, user already have data in the "ratio.dat". And this data in the old format, i.e. $rat["upload"] in MiB.
What he will have at this point? MiB instead GiB?

Reply:
The old data is currently setup to be whipped and replaced with new data when they upgrade the plugin.

If this is a problem, I can migrate the data to the new configuration by adding anther function to `ratio.php`.

Reply:
I added a new function to migrate the configuration to the new version of the plugin.